### PR TITLE
Adding more options to the BOT (suppor us ,ethiopian and world statstics buttons

### PR DESCRIPTION
**Pull request by dawondyifraw**
… statistics , what should i do and Self diagnosis texts..to be developed further to push data to  whats and facebook bot integration next

### CODE REVIEW

**Comment by theShinigami** [comment]
yeah, we need a solid stat server to get the data

**Comment by dawondyifraw** [comment]
let me send you a good API link to use ? For the Ethiopian case i see two options.
1. lets build an API our selves to push data through some telegram bot or you can suggest

**Comment by theShinigami** [comment]
check https://covid19.locale.ai/, find out if they got some stat api

**Comment by Eyob-T** [comment]
check this https://github.com/ExpDev07/coronavirus-tracker-api for an API. Its used by the above mentioned repo

**Comment by dadenewyyt** [comment]
@dotmike added viber bot on the last PR

**Comment by dawondyifraw** [comment]
> check https://covid19.locale.ai/, find out if they got some stat api

this not latest still now it says 6 cases for ethiopia :)

File: src/Secrets.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const MAIN_CHANNEL_USERNAME = "";
 const ADMINS = []
 
 
-const BOT_TOKEN = "819778324:AAEsM_p4hZbizY3zwz75uqhQooKBaMXSjMw";
```
Comment (brookmg) [inline src/Secrets.js:20]: + Token should be hidden

**Comment by brookmg** [comment]
I think you are putting a lot in one PR @dadenewyyt ... best if you split it up

File: src/Secrets.js (hunk 1 above)
Comment (theShinigami) [reply src/Secrets.js:20]: It already been revoked.

**Comment by Eyob-T** [comment]
> > check https://covid19.locale.ai/, find out if they got some stat api
> 
> this not latest still now it says 6 cases for ethiopia :)

Found this accurate reporting dashboard  .... trying to find API they used 
https://www.bing.com/covid

**Comment by theShinigami** [comment]
Here is the api url https://www.bing.com/covid/data?IG=A37A826E6B19477AA1576E0E96D4F0E8

File: src/Secrets.js (hunk 1 above)
Comment (brookmg) [reply src/Secrets.js:20]: Ya I just noticed. Someone should merge your PR I think

**Comment by dadenewyyt** [comment]
@brookmg who the fuck are you telling what to do ? Are the fucking moorom project manger ?or what ? Dont fucking say a word again in any channel

**Comment by brookmg** [comment]
@dadenewyyt I am not the project manager but other people might find it hard to contribute if you add alot at once. It will make it harder to review also ( to whomever is going to do that ). Have some decency!  cc:@dotmike

**Comment by dadenewyyt** [comment]
@brookmg I m telling you not teach me , you are not telling me to do anything. You useless mooron. Just shut your mouth and help your self. I didnt ask for your help.

**Comment by brookmg** [comment]
Damn you are toxic, I bet you never worked on a public stuff. I am not going to engage in this

**Comment by dadenewyyt** [comment]
@brookmg go fuck your self you piece of shit. What have you done ? get of your ass and don ever come back to me again.

**Comment by theShinigami** [comment]
We're in the same team man, calm down!

**Pull request closed without merging**